src/x11_init.c

```diff
@@ -715,7 +715,7 @@ static GLFWbool initExtensions(void)
                                 &_glfw.x11.randr.minor))
             {
                 // The GLFW RandR path requires at least version 1.3
-                if (_glfw.x11.randr.major > 1 || _glfw.x11.randr.minor >= 3)
+                if (_glfw.x11.randr.major >= 1 && _glfw.x11.randr.minor >= 3)
```

Comment:
Going by the comment *only*, the major version check should be greater than or equal to 1 and not just greater than 1, and the logical operator should be AND (`&&`) and not OR (`||`) between the major and minor version checks.